src/ucp/rndv/rndv_get.c

```diff
@@ -208,6 +208,32 @@ static void ucp_rndv_get_zcopy_proto_abort(ucp_request_t *request,
     }
 }
 
+static void ucp_rndv_get_zcopy_proto_reset(ucp_request_t *request)
+{
+    if (!(request->flags & UCP_REQUEST_FLAG_PROTO_INITIALIZED)) {
+        return;
+    }
+
+    request->flags &= ~UCP_REQUEST_FLAG_PROTO_INITIALIZED;
+    ucp_proto_request_check_reset_state(request);
```

Comment:
why are we using this function only from ucp_rndv_get_zcopy_proto_reset and not other protocols?

Reply:
All basic protocols are checked in `ucp_proto_request_restart`, rndv get has several stages and super request, so I decided added extra check. @yosefe Do you think it's redundant here or need to move common check to every `*_reset` function?

Reply:
why the check in ucp_proto_request_restart() is not enough for rndv?

Reply:
I believe it should be enough, added just because rndv looks more complex, will remove then .